packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -281,6 +288,23 @@ export class ApexReplayDebug extends LoggingDebugSession {
             this.convertClientLineToDebugger(lineArg)
           );
         }
+        // If there is a condition and that condition is a checkpoint then
+        // sent CHECKPOINT_INFO_EVENT with the args.source, line and uri
+        if (args.breakpoints[i].condition) {
```

Comment:
I just remember - search for `supportsConditionalBreakpoints` in this file and flip it to true.